{
  "abstract": [
    {
      "text": "Names of exceptions raised by ",
      "type": "text"
    },
    {
      "code": "NSInvocationOperation",
      "type": "codeVoice"
    },
    {
      "text": " if there is an error when calling the ",
      "type": "text"
    },
    {
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/1543615-result",
      "isActive": true,
      "type": "reference"
    },
    {
      "text": " method.",
      "type": "text"
    }
  ],
  "documentVersion": 0,
  "hierarchy": {
    "paths": [
      [
        "doc://com.apple.documentation/documentation/technologies",
        "doc://com.apple.documentation/documentation/foundation",
        "doc://com.apple.documentation/documentation/foundation/task_management",
        "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation"
      ],
      [
        "doc://com.apple.documentation/documentation/technologies",
        "doc://com.apple.documentation/documentation/foundation",
        "doc://com.apple.documentation/documentation/foundation/processes_and_threads",
        "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation"
      ]
    ]
  },
  "identifier": {
    "interfaceLanguage": "occ",
    "url": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/result_exceptions"
  },
  "kind": "symbol",
  "legacy_identifier": 1661425,
  "legalNotices": {
    "copyright": "Copyright &copy; 2025 Apple Inc. All rights reserved.",
    "privacyPolicy": "https://www.apple.com/privacy/privacy-policy",
    "termsOfUse": "https://www.apple.com/legal/internet-services/terms/site.html"
  },
  "metadata": {
    "modules": [
      {
        "name": "Foundation"
      }
    ],
    "role": "collectionGroup",
    "roleHeading": "API Collection",
    "title": "Result Exceptions"
  },
  "references": {
    "doc://com.apple.documentation/documentation/foundation": {
      "identifier": "doc://com.apple.documentation/documentation/foundation",
      "kind": "symbol",
      "role": "collection",
      "title": "Foundation",
      "type": "topic",
      "url": "/documentation/foundation"
    },
    "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation",
      "kind": "symbol",
      "role": "symbol",
      "title": "NSInvocationOperation",
      "type": "topic",
      "url": "/documentation/foundation/nsinvocationoperation"
    },
    "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/1543615-result": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/1543615-result",
      "kind": "symbol",
      "role": "symbol",
      "title": "result",
      "type": "topic",
      "url": "/documentation/foundation/nsinvocationoperation/1543615-result"
    },
    "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationcancelledexception": {
      "abstract": [
        {
          "text": "The name of the exception raised if the ",
          "type": "text"
        },
        {
          "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/1543615-result",
          "isActive": false,
          "type": "reference"
        },
        {
          "text": " method is called after the operation was cancelled. ",
          "type": "text"
        }
      ],
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationcancelledexception",
      "kind": "symbol",
      "role": "symbol",
      "title": "NSInvocationOperationCancelledException",
      "type": "topic",
      "url": "/documentation/foundation/nsinvocationoperationcancelledexception"
    },
    "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationvoidresultexception": {
      "abstract": [
        {
          "text": "The name of the exception raised if the ",
          "type": "text"
        },
        {
          "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperation/1543615-result",
          "isActive": false,
          "type": "reference"
        },
        {
          "text": " method is called for an invocation method with a ",
          "type": "text"
        },
        {
          "code": "void",
          "type": "codeVoice"
        },
        {
          "text": " return type. ",
          "type": "text"
        }
      ],
      "identifier": "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationvoidresultexception",
      "kind": "symbol",
      "role": "symbol",
      "title": "NSInvocationOperationVoidResultException",
      "type": "topic",
      "url": "/documentation/foundation/nsinvocationoperationvoidresultexception"
    },
    "doc://com.apple.documentation/documentation/foundation/processes_and_threads": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/processes_and_threads",
      "kind": "article",
      "role": "collectionGroup",
      "title": "Processes and Threads",
      "type": "topic",
      "url": "/documentation/foundation/processes_and_threads"
    },
    "doc://com.apple.documentation/documentation/foundation/task_management": {
      "identifier": "doc://com.apple.documentation/documentation/foundation/task_management",
      "kind": "article",
      "role": "collectionGroup",
      "title": "Task Management",
      "type": "topic",
      "url": "/documentation/foundation/task_management"
    },
    "doc://com.apple.documentation/documentation/technologies": {
      "identifier": "doc://com.apple.documentation/documentation/technologies",
      "kind": "technologies",
      "title": "Technologies",
      "type": "topic",
      "url": "/documentation/technologies"
    }
  },
  "schemaVersion": {
    "major": 0,
    "minor": 3,
    "patch": 0
  },
  "sections": [],
  "topicSections": [
    {
      "anchor": "1673904",
      "identifiers": [
        "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationvoidresultexception",
        "doc://com.apple.documentation/documentation/foundation/nsinvocationoperationcancelledexception"
      ],
      "kind": "taskGroup",
      "title": "Constants"
    }
  ],
  "variants": [
    {
      "paths": [
        "documentation/foundation/nsinvocationoperation/result_exceptions"
      ],
      "traits": [
        {
          "interfaceLanguage": "occ"
        }
      ]
    }
  ]
}
